tool/popularities.js

```diff
@@ -12,14 +12,25 @@
 const fs = require("fs");
 
 const csv = require("@fast-csv/parse");
+const got = require("got");
+
+const CURRENT_URL =
+  "https://mdn-popularities-prod.s3.amazonaws.com/current.txt";
```

Comment:
Is there anything you can do to change that to a text/plain?

Reply:
👍